config_manager/include/config_manager.h

```diff
@@ -0,0 +1,16 @@
+//
+// Created by viacheslav on 11.04.23.
+//
+
+#ifndef MOTORLAND_FORK_CONFIG_MANAGER_H
```

Comment:
According to [guidelines](https://github.com/Moleus/motorland/blob/7adb5f895c54bff638100a9fa85536c008456044/docs/eng/contributing.md?plain=1#L21) we use `#pragma once` in the code.